DEVELOPING.md

```diff
@@ -49,6 +49,7 @@ Within a development environment:
 1. **Build and test the project**. You will usually use `colcon`.
 
     ```bash
+    . /opt/ros/${ROS_DISTRO}/setup.bash
```

Comment:
Do we need to add this to the "Update package dependencies" section, so that both examples are self-contained?

Reply:
Fixed!